packages/dito/app/routes/vorpruefung.einschaetzung.tsx

```diff
@@ -0,0 +1,96 @@
+import Box from "@digitalcheck/shared/components/Box";
+import Button from "@digitalcheck/shared/components/Button";
+import ButtonContainer from "@digitalcheck/shared/components/ButtonContainer";
+import Container from "@digitalcheck/shared/components/Container";
+import { type LoaderFunctionArgs } from "@remix-run/node";
+import { Form, redirect } from "@remix-run/react";
+import { getAnswersFromCookie } from "cookies.server";
+import { assessment } from "resources/content";
+import { PATH_RESULT } from "resources/staticRoutes";
+
+export async function action({ request }: LoaderFunctionArgs) {
+  const { answers } = await getAnswersFromCookie(request);
+
+  const body = await request.formData();
+  const { _action, ...values } = Object.fromEntries(body);
+
+  const pdfValues = { ...values, ...answers, download: true };
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-explicit-any
+  const queryParams = new URLSearchParams(pdfValues as any).toString();
+
+  // TODO:
+  // if (_action === "receiveEmail") {
+  // }
+
+  if (_action === "downloadPdf") {
+    return redirect(`digitalcheck-vorpruefung.pdf?${queryParams}`);
+  }
+
+  return true;
+}
+
+export default function Assessment() {
+  return (
+    <>
+      <Container paddingBottom="0">
+        <Box
+          heading={{
+            tagName: "h1",
+            text: assessment.title,
+            look: "ds-heading-02-reg",
+          }}
+          content={{
+            markdown: assessment.subtitle,
+          }}
+        ></Box>
+      </Container>
+      <Container paddingBottom="48">
+        <Form method="post" reloadDocument>
```

Comment:
Would this have worked with a `type=submit` button instead of `reloadDocument`?

Reply:
`reloadDocument` is needed specifically to force the browser to download the file... it's a special case I found in the Remix docs somewhere...